src/PrivateKey.ts

```diff
@@ -11,8 +11,7 @@ export class PrivateKey {
     constructor(private key: Key) {}
 
     /** Instantiate private key from an `elliptic`-format private key */
-    public static fromElliptic(privKey: ec.KeyPair, keyType = KeyType.k1): PrivateKey {
-        const privArray = privKey.getPrivate().toArray();
+    public static fromElliptic(privKey: ec.KeyPair, keyType: KeyType): PrivateKey {
```

Comment:
Note that removing the default from this argument makes this PR a breaking change.

Reply:
This file isn't in v20